presto-client/src/main/java/com/facebook/presto/client/OkHttpUtil.java

```diff
@@ -186,6 +191,30 @@ public static void setupSsl(
         }
     }
 
+    private static void validateCertificates(KeyStore keyStore)
```

Comment:
Would it be possible to add unit test for that?

Reply:
I wasn't able to make `openssl` to generate expired certificates. We know it works in the positive case (based on integration tests) and it worked when I tested manually. Since this is only to improve the error message, let's revisit when we do CLI integration tests for certificate auth.

Reply:
Did you try generate a cert valid for 1 day and wait 1 day till it expires?

Reply:
@findepi That's a good idea. I'll do that now and put up a new PR with tests after it is expired.

Reply:
According to my Google searches, the `-days` option used to support negative days, but I guess they added validation at some point...